tests/test_data/core/core__count_encounter_month.txt

```diff
@@ -1,57 +1,57 @@
-(10, '2018-07-01', 'ambulatory', None, 'male', None, 'not hispanic or latino')
```

Comment:
This was regenned more than once during development, but should be exactly the same, just sorted differently. I should probably fix this quirk of the reference data export mechanism